mirdata/rwc_classical.py

```diff
@@ -287,14 +287,14 @@ def track_ids():
 def load(data_home=None):
```

Comment:
this file should not be included in this PR

Reply:
Black modifies it.

Reply:
try running it as defined in CONTRIBUTING
`black --target-version py37 --skip-string-normalization mirdata/`
as a rule of thumb try avoiding global commits and refer to specific files.
you can discard the commits on a specific file by updating the master and replacing your version of the file with the version from the master 
`git checkout master file_to_restore.ext`